src/shared/components/annotation/styles/oncokb/collapsible.module.scss

```diff
@@ -0,0 +1,18 @@
+@import "vars.module";
+
+$collapsible-height: 30px;
+$collapsible-header-color: white;
+
+.collapsible-header {
```

Comment:
when these are applied, i would think that the hyphen in style.collapsible-height would cause javascript to error, no?

Reply:
Yeah, I need to use style['collapsible-height'] which was ok because the IDE autogenerated. But I would open to change to style.collapsibleHeight if this is how we should do it and I love to keep the consistency.